components/SettingsControls/samples/SettingsExpanderSample.xaml.cs

```diff
@@ -4,7 +4,6 @@
 
 namespace SettingsControlsExperiment.Samples;
 
-[ToolkitSampleBoolOption("IsCardEnabled", true, Title = "Is Enabled")]
```

Comment:
Maybe just comment this out and leave a comment with a link to the issue so we remember to add it back when that issue is fixed?